src/pybind/mgr/dashboard/controllers/__init__.py

```diff
@@ -368,7 +375,7 @@ def generate_routes(url_prefix):
 
 
 def json_error_page(status, message, traceback, version):
-    cherrypy.response.headers['Content-Type'] = 'application/json'
+    cherrypy.response.headers['Content-Type'] = 'application/vnd.ceph.api.v1.0+json'
```

Comment:
On error conditions, I'm not sure if we do really need to return the exact same `content-type` as the `Accept` header...

For example, for the nginx webserver [they closed as invalid a request of this behaviour](https://trac.nginx.org/nginx/ticket/1818). This is a complex part of the [HTTP standard](https://www.w3.org/Protocols/rfc2616/rfc2616-sec12.html), but to summarize it: Accept header is just a hint and the server is allowed to respond with a different `Content-Type`, especially for error responses.

In our case, we may think that `application/vnd.ceph.api.v1.0+json` is just a subset of `application/json` MIME types. Ideally the browser should send something like this:
```
Accept: application/vnd.ceph.api.v1.0+json, application/json
```

Reply:
Shouldn't it be 'Content-Type'? (instead of Accept)

Reply:
Yes, what I meant is that usually the server should honor (Content-Type) what the client requested (Accept), but in case of error I don't think it's mandatory based on the references pointed.

Reply:
Okay, so if I understand it correctly, we need to specify both application/*+json & application/json for response Content-Type headers, right?

Reply:
No need of. Just returning the `application/json` is fine for regular/plain JSON pages, like the error ones.